### DISCUSSION TITLE
No "Users" section in "NetBox Administration" (no place to add standard user)

### DISCUSSION THREAD

**Discussion by andrei-korshikov** [netbox-community/netbox#14003] [Help Wanted!] [answered]
Quite strange, but I have only "Python social auth" and "System" sections in my "NetBox Administration". I mean, no "Authentication and Authorization", no "Extras" and no "Users". I installed NetBox manually (not using Docker), so I suspect I'm missing some Python modules or system libraries. What do you recommend for further investigation? NetBox v3.6.3.

**Comment by abhi1693** [comment, marked as answer]
Look at the frontend, here is a sample https://demo.netbox.dev/users/users/

**Reply by andrei-korshikov** [reply]
Shame on me:( I overlooked this "Admin" section at the very bottom of the main (left) menu. @abhi1693 thanks a lot!
